### PR TITLE
Securityhudicons

### PR DESCRIPTION
**Pull request by NanakoAC**
This is now updated to baymerge and completed

- rscadd "Overhauled all of the security hud job icons, to be more visually pleasing and better communicate roles"
- bugfix "Intern positions and psychiatrists now have an icon. All jobs that were missing an icon should be fixed"
- tweak "Made the loyalty implant icon a less obtrusive green light, instead of a big red square"
- tweak "CCIA, ERT, IAA and any similar nanotrasen representatives now have the N logo as their icon"

### CODE REVIEW

File: html/changelogs/Nanako-PR-432.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd "Overhauled all of the security hud job icons, to be more visually pleasing and better communicate roles"
```
Comment (skull132) [inline html/changelogs/Nanako-PR-432.yml]: Missing colons here, I think?

**Comment by skull132** [comment]
Merge error, most likely to do with modifications around hud.dmi.

**Comment by NanakoAC** [comment]
Welp, as far as i can tell the latest byond update has broken this by scrwing up my icons, ill test an experimental fix, might take a while to get around to it, leave this DNM for now

**Comment by SierraKomodo** [comment]
Does this include a separate icon for the 'Visitor' role or does that still use the Assistant 'A'? I'd recommend a separate icon due to the IC differences between the two roles.

**Comment by skull132** [comment]
Can you post preview images real quick, of how it looks like on a person?

**Comment by skull132** [comment]
https://puu.sh/t0MSf/84fe88087a.png

I'd say they're too big? You also have to take into account the fun fact that these icons add to the click area of a person. Right now, they effectively increase it by 2/3rds, as a 2 tall stack is about 2/3rds the size of a human mob.

**Comment by Alberyk** [comment]
They are indeed too big, I think that new ones for the missing icons are a great idea, but, just keep the size they are atm.

**Comment by NanakoAC** [comment]
making the icons smaller is not possible. I already did that, ive shrunk them several times
The total area of the icons is increased from 8x8 to 12x12, and ive used all kinds of tricks to try to keep them smaller. Its just not really possible to make a meaningful design in any smaller space

Many, like the paramedic, or psych, are on a dangerously thin line of detail making as much use of blurred pixels as possible, they degrade into an unrecogniseable mess any smaller. Many others, especially the security shields, can't maintain symmetry. bars on the warden icon lose the gaps inbetween, etc
The only thing left i can really do is remove the 1 pixel border, but i dont think that would look good

**Comment by SierraKomodo** [comment]
Is there a reason why we're not just sticking with the simplistic style used in the current hud icons?

**Comment by NanakoAC** [comment]
because they're awful and convey almost nothing about the job, and quite a few of them didnt even have icons

I have a couple more ideas for optimising them, lemme see what i can do

**Comment by LordFowl** [comment]
@NanakoAC They are very large, and they become unbearably large the more status mods get added. I suggest making stuff like loyalty implants, arrest status, etcetera overlay directly onto the main job indicator, perhaps as a translucent coloured line. Because the way it is now easily 1/3rd of the available space can become occupied by a sufficiently notable crewmember.

**Comment by NanakoAC** [comment]
working on removing borders to reduce size, dnm for a lil while

**Comment by NanakoAC** [comment]
The icons are now 10x10 with 80% opacity, the little flashy lights are anchored to it

**Comment by Arrow768** [comment]
Do you have any pictures ?

**Comment by skull132** [comment]
Changes required:
- Remove the opacity
- Move the arrest and detainment status indicators up, so they're right against the job indicators

**Pull request merged**